internal/checkers/helpers_string.go

```diff
@@ -0,0 +1,11 @@
+package checkers
+
+import (
+	"go/ast"
+
+	"golang.org/x/tools/go/analysis"
+)
+
+func isStringsContainsCall(pass *analysis.Pass, ce *ast.CallExpr) bool {
```

Comment:
`string` in `helpers_string.go` is too common naming

let's move it into `helpers_pkg_func.go` near `isRegexpMustCompileCall`